### PR TITLE
Add the leak_vm meta-data in add-to-app sample

### PR DESCRIPTION
**Pull request by eggfly**
Add an example code for custom switch leak VM settings, issue link:
* https://github.com/flutter/flutter/issues/95903

Related PR:
* https://github.com/flutter/engine/pull/30541

## Pre-launch Checklist

- [x] I read the [Flutter Style Guide] _recently_, and have followed its advice.
- [x] I signed the [CLA].
- [x] I read the [Contributors Guide].
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] All existing and new tests are passing.

### CODE REVIEW

File: add_to_app/multiple_flutters/multiple_flutters_android/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -37,6 +37,15 @@
             android:hardwareAccelerated="true"
             android:windowSoftInputMode="adjustResize"
             />
+        <!-- Whether leave or clean up the VM after the last shell shuts down. If your want to let
+             your app destroy the last shell and re-create shells more quickly, set it to true,
+             otherwise if you want to clean up the memory of the leak VM, set it to false. This
+             setting should works after the Flutter engine supports custom switch. View this link:
```
Comment (domesticmouse) [inline add_to_app/multiple_flutters/multiple_flutters_android/app/src/main/AndroidManifest.xml:43]: When will this custom switch in engine make it to stable?

Comment (blasten) [reply add_to_app/multiple_flutters/multiple_flutters_android/app/src/main/AndroidManifest.xml:43]: after 2.10, which is the next stable.

**Review by blasten — COMMENTED** [review]
worth adding a comment about the flutter version where this will be enabled

File: add_to_app/multiple_flutters/multiple_flutters_android/app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (eggfly) [reply add_to_app/multiple_flutters/multiple_flutters_android/app/src/main/AndroidManifest.xml]: Done. Add a 2.10 stable version comment. cc @blasten @domesticmouse  Thanks 😄

**Review by RedBrogdon — APPROVED** [review]

**Comment by RedBrogdon** [comment]
This looks ready to land, though I'd like to make sure I completely understand the change. 😄 

I've read through https://github.com/flutter/flutter/issues/95903, and it looks like setting the flag to false will cause the Android embedder to avoid leaking the VM into the overall process, making it possible for the VM to be fully shut down while the app is running.

Is there a reason why it makes sense to set this flag only in the "multiple Flutters" example? Is it due to the fact that this is the only sample that manually spins up and shuts down engine instances over a single run?

**Comment by eggfly** [comment]
> This looks ready to land, though I'd like to make sure I completely understand the change. 😄
> 
> I've read through [flutter/flutter#95903](https://github.com/flutter/flutter/issues/95903), and it looks like setting the flag to false will cause the Android embedder to avoid leaking the VM into the overall process, making it possible for the VM to be fully shut down while the app is running.
> 
> Is there a reason why it makes sense to set this flag only in the "multiple Flutters" example? Is it due to the fact that this is the only sample that manually spins up and shuts down engine instances over a single run?

Yes, I think it's the only one.

**Pull request merged**